packages/calypso-e2e/src/lib/pages/editor-page.ts

```diff
@@ -69,9 +71,10 @@ export class EditorPage {
 		this.editorSettingsSidebarComponent = new EditorSettingsSidebarComponent( page, this.editor );
 		this.editorPublishPanelComponent = new EditorPublishPanelComponent( page, this.editor );
 		this.editorNavSidebarComponent = new EditorNavSidebarComponent( page, this.editor );
+		this.editorListViewComponent = new EditorListViewComponent( page, this.editor );
 	}
 
-	/* Generic methods */
+	//#region Generic and Shell Methods
```

Comment:
This class will be full of many small methods. I figured that adding collapsible regions would be worth it for tidiness.